docs/Doxyfile

```diff
@@ -0,0 +1,21 @@
+DOXYFILE_ENCODING      = UTF-8
+
+PROJECT_NAME           = "OpenTelemetry C++ API/SDK"
```

Comment:
I wonder if we should have separate documentation for the API and SDK. Both are for different audiences, and it might be confusing to have them intermingled.

Reply:
Yes, I am working on this, too.